### PR TITLE
Compatibility with Firefox

### PR DESCRIPTION
**Pull request by flcdrg**
These changes allow the extension to work with Firefox

- Also resolves a couple of warnings/errors I saw when running the Firefox debugger

### CODE REVIEW

**Comment by mjisaak** [comment]
Hi David, 

thanks a lot for your work. I will try to publish the Extension to the Firefox Marketplace and then gladly accept the PR.

**Comment by flcdrg** [comment]
No worries! I have some more ideas to contribute. Will send those through separately when I get a chance.

File: src/manifest.json [hunk 1]
```diff
@@ -16,11 +16,18 @@
   // "options_page": "options.html", // for debug
   "permissions": [
     "contextMenus",
-    "storage"
+    "storage",
+    "clipboardWrite"
```
Comment (mjisaak) [inline src/manifest.json:20]: @flcdrg Do we need `clipboardWrite` here?

**Review by mjisaak — COMMENTED** [review]
I successfully published the extension to the Firefox marketplace: 
https://addons.mozilla.org/en-US/firefox/addon/mvp-docs-learn-champion/

**Comment by flcdrg** [comment]
I found that I did, otherwise the paste to the clipboard didn't work. Firefox shows this error in the console without it:

image

This seems odd as the function seems to fit the definition of a "short-lived event handler for a user action"

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard

**Pull request merged**